services/edit/db.js

```diff
@@ -247,6 +247,17 @@ function getHTML(uri) {
 
 /**
  * @param {string} uri
+ * @param {string} queries
+ * @return {Promise}
+ */
+function getHTMLWithQuery(uri, queries) {
+  var queryUrl = uri + extHtml + editMode + queries;
```

Comment:
queries should probably be an object, `_.assign`'d to `{ edit: true}`, then stringified into `?edit=true&query1=whatever&query2=whatever` so consumers don't need to care about the ampersands/question marks